runtime/j9vm/jvm.c

```diff
@@ -74,6 +74,65 @@
 #include <unistd.h>
 #endif /* defined(AIXPPC) */
 
+#if defined(J9ZOS390)
+#include <stdlib.h>
+
+/*
+ * These offsets and constants are found in "Language Environment Vendor Interfaces", downloaded from
+ * https://www.ibm.com/servers/resourcelink/svc00100.nsf/pages/zosV2R4SA380688/$file/ceev100_v2r4.pdf
+ */
+#if defined(J9VM_ENV_DATA64)
+#define OFFSET_CAA_CEDB 0x348
+#define OFFSET_CEDB_LANG 0x18
+#else /* defined(J9VM_ENV_DATA64) */
+#define OFFSET_CAA_CEDB 0x218
+#define OFFSET_CEDB_LANG 0x10
+#endif /* defined(J9VM_ENV_DATA64) */
+
+/* An enclave data block should begin with this eye-catcher. */
+#pragma convlit(suspend)
+static const char cedb_eye[4] = "CEDB";
+#pragma convlit(resume)
+
+/*
+ * Determine whether the mainline language is C/C++.
+ */
+static BOOLEAN
+mainlineLanguageIsC(void)
+{
+	BOOLEAN result = FALSE;
+	/* Locate the common anchor area. */
+	const char *caa = (const char *)_gtca();
+	/* Fetch the pointer to the C/C++ environment data block. */
+	const char *cedb = *(const char * const *)(caa + OFFSET_CAA_CEDB);
+
+	/* Do some basic checking of the cedb pointer. */
+	if ((NULL !=  cedb) && (0 == memcmp(cedb, cedb_eye, sizeof(cedb_eye)))) {
```

Comment:
Can it be? There is an extra space in `NULL != cedb`

Reply:
Fixed in 9762f394c3eccbb2511ff93f45f058105b43afdf.